spec/features/user_can_add_reviews_spec.rb

```diff
@@ -18,3 +18,22 @@
     )
   end
 end
+
+feature 'reviews paginate' do
+  let(:game) { FactoryGirl.create(:game) }
+  let(:user) { FactoryGirl.create(:user) }
+
+  scenario 'reviews show on multiple pages' do
+    sign_in_as(user)
+    8.times { make_review_for_game(game) }
+    visit game_path(game)
+    click_link "review this game"
+    fill_in "your review", with: "something something different"
+    select 5, from: "rating"
+    click_button "add review"
+    visit game_path(game)
+    expect(page).not_to have_content("something something different")
+    click_link("Next")
+    expect(page).to have_content("something something different")
```

Comment:
Also, you don't have to create a review for your games to check pagination. You just want a bunch of games and make sure that if you click "Next", you still see another game.